### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by r4lv**

### CODE REVIEW

**Comment by r4lv** [comment]
I just removed the `xloci`-commit, it does not belong here.

File: vip_hci/var/__init__.py [hunk 1]
```diff
@@ -9,5 +9,5 @@
 
 from .filters import *
 from .fit_2d import *
-from .shapes import *
 from .plotting import *
+from .shapes import *
```
Comment (carlgogo) [inline vip_hci/var/__init__.py]: what is the purpose of this change in ordering?

Comment (r4lv) [reply vip_hci/var/__init__.py]: That is to order the imports alphabetically, like in the other submodules.
